docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1673,6 +1673,13 @@ Default stratifications to all observers should include scenario and input draw.
     - Default
     - Default
     - Default
+  * - 20.3
```

Comment:
Decided to make this a separate run so that Zeb could do 20.2 task-shifted and engineering could pick this up in advance of the neonatal mortality run (although we may have to push this and neonatal mortality until after the hemoglobin (and maybe pregnancy) refactors depending on how it shakes out.